plugin/storage/integration/elasticsearch_test.go

```diff
@@ -52,6 +54,9 @@ type ESStorageIntegration struct {
 	client   *elastic.Client
 	v8Client *elasticsearch8.Client
 
+	ArchiveSpanReader spanstore.Reader
+	ArchiveSpanWriter spanstore.Writer
```

Comment:
where do we initialize these?

Reply:
https://github.com/jaegertracing/jaeger/blob/main/plugin/storage/integration/elasticsearch_test.go#L146-L149

Reply:
could we move that directly into the archive test?

Reply:
we can leave it here